pkg/cmd/sshpatch/export_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sshpatch
+
+import (
+	"context"
+
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/ssh"
+	"github.com/gardener/gardenctl-v2/pkg/target"
+)
+
+type TestUserBastionListPatcherImpl struct {
+	userBastionListPatcherImpl
+}
+
+func NewTestUserBastionPatchLister(manager target.Manager) *TestUserBastionListPatcherImpl {
```

Comment:
Should not be in export_test

Reply:
Move it to the `sshpatch_suite_test` instead

Reply:
Should `NewTestOptions` then moved as well?